QuadrupedalRobots/ETGRL/deployment/robots/minitaur.py

```diff
@@ -0,0 +1,1435 @@
+# coding=utf-8
```

Comment:
Is minitaur*.py needed? Are this codes for minitaur robots?

Reply:
it's needed, the motion imitation modify the code from minitaur, it's parent class.